_includes/sections/video-streaming.html

```diff
@@ -0,0 +1,27 @@
+<h1 id="Video Streaming" class="anchor"><a href="#video"><i class="fas fa-link anchor-icon"></i></a> Video Streaming</h1>
+
+  <div class="row mb-2">
+
+    {% include card.html color="success"
+    title="MediaGoblin"
+    image="/assets/img/tools/mediagoblin.png"
+    url="https://www.mediagoblin.org/"
+    footer="OS: Windows, macOS, Linux, FreeBSD, Web."
+    description="MediaGoblin is a free/libre tool to host media files in a distributed manner."
+    %}
+
+    {% include card.html color="primary"
+    title="Invidious"
+    image="/assets/img/tools/invidious.png"
+    url="https://github.com/omarroth/invidious/wiki/Invidious-Instances"
+    footer="OS: Windows, macOS, Linux, Web."
+    description="Invidious, an open-source privacy respecting API for YouTube."
```

Comment:
```suggestion
    description="Invidious is an open-source alternative front-end to YouTube. It has no ads and no need for a Google account to save subscriptions. <a href="https://github.com/omarroth/invidious/wiki/Invidious-Instances">List of Instances</a>"
```

Invidious isn't just an API; it's also a front-end for YouTube as described by the description and the README of the repo: https://github.com/omarroth/invidious. Also, just like how we list instances for searx, we should do the same for Invidious.